python/src/aiconfig/editor/client/src/components/global/ShareButton.tsx

```diff
@@ -29,9 +31,12 @@ export default memo(function ShareButton({ onShare }: Props) {
     openModal();
   };
 
+  // We use 'Notebook' in Gradio for GradioNotebook packaging
+  const artifactName = mode === "gradio" ? "Notebook" : "Workbook";
```

Comment:
Should it always just be "Notebook"? Nbd, can fix forward later if we want

Reply:
We're using workbook terminology in vs code already